app/services/prototype.session.server.ts

```diff
@@ -1,8 +1,14 @@
 import { parse } from "cookie";
 import { createCookieSessionStorage, redirect } from "react-router";
 import { serverConfig } from "~/config/config.server";
+import { ServicesContext } from "~/services/prototype.servicesContext.server";
 import type { AuthenticationContext } from "./prototype.oAuth.server";
-import { ServicesContext } from "./prototype.servicesContext.server";
+
+const getSecret = () => {
+  return serverConfig().ENVIRONMENT === "development"
```

Comment:
We can use the public config instead:
```suggestion
  return config().ENVIRONMENT === "development"
```